tidb-cloud/ticloud-import-start.md

```diff
@@ -69,31 +69,34 @@ ticloud serverless import start --source-type AZURE_BLOB --azblob.uri <azure-blo
-| --csv.backslash-escape           | Specifies whether to parse backslash inside fields as escape characters in a CSV file. The default value is `true`.        | No       | Only works in non-interactive mode.                  |
-| --csv.delimiter string           | Specifies the delimiter used for quoting a CSV file. The default value is `\`.                                             | No       | Only works in non-interactive mode.                  |
-| --csv.separator string           | Specifies the field separator in a CSV file. The default value is `,`.                                                     | No       | Only works in non-interactive mode.                  |
-| --csv.skip-header                | Specifies whether the CSV file contains a header line.                                                                     | No       | Only works in non-interactive mode.                  |
-| --csv.trim-last-separator        | Specifies whether to treat the separator as the line terminator and trim all trailing separators in a CSV file.            | No       | Only works in non-interactive mode.                  |
-| --csv.not-null                   | Specifies whether a CSV file can contain any NULL values.                                                                  | No       | Only works in non-interactive mode.                  |
-| --csv.null-value string          | Specifies the representation of NULL values in the CSV file. (default "\\N")                                                       | No       | Only works in non-interactive mode.                  |
-| -h, --help                       | Shows help information for this command.                                                                                   | No       | Works in both non-interactive and interactive modes. |                                                                           | No       | Works in both non-interactive and interactive modes. |
+| --s3.uri string                  | Specifies the S3 URI in `s3://<bucket>/<path>` format. Required when source type is S3.                                                       | Yes      | Only works in non-interactive mode.                  |
+| --oss.access-key-id string       | Specifies the AccessKey ID of Alibaba OSS.                                                                                          | No       | Only works in non-interactive mode.                  |
+| --oss.access-key-secret string   | Specifies the AccessKey Secret of Alibaba OSS.                                                                                      | No       | Only works in non-interactive mode.                  |
+| --oss.uri string                 | Specifies the OSS URI in `oss://<bucket>/<path>` format. Required when the source type is OSS.                                                    | Yes      | Only works in non-interactive mode.                  |
```

Comment:
Not sure about this change, PTAL

```suggestion
| --oss.uri string                 | Specifies the OSS URI in `oss://<bucket>/<path>` format. Required when the source type is Alibaba OSS.                                                    | Yes      | Only works in non-interactive mode.                  |
```

Reply:
<img width="472" alt="image" src="https://github.com/user-attachments/assets/e6e4df0b-a9a6-441e-b2eb-ab12b6d4b18e" />
maybe use OSS is better.